### ISSUE TITLE
[iceprod2] distributed databases require timing info (Trac #1365)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1357]
In order to sync properly, every row in every table needs to have a timestamp associated with it (at least if we are the master). Then we can do out of order updating by just taking the last one and dropping (or overwriting) the others.

This is important if one site gets cut off from the master for a while and can't sync immediately.  The buffer will store these updates, and then try applying them later.  But if the master has moved on and assigned the task to another grid, we don't want that update.

Note that this adds about 50 bytes for every row. So if we have 1B rows, that's 50GB of overhead. Yay?

Migrated from https://code.icecube.wisc.edu/ticket/1365
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "In order to sync properly, every row in every table needs to have a timestamp associated with it (at least if we are the master). Then we can do out of order updating by just taking the last one and dropping (or overwriting) the others.\n\nThis is important if one site gets cut off from the master for a while and can't sync immediately.  The buffer will store these updates, and then try applying them later.  But if the master has moved on and assigned the task to another grid, we don't want that update.\n\nNote that this adds about 50 bytes for every row. So if we have 1B rows, that's 50GB of overhead. Yay?", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] distributed databases require timing info", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-09-22T20:40:22", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151002T21:59:27: 
* david.schultz commented:

> In changeset 138206:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138206"
> add master_update_history table and indexes. closes #1357
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
